### PR TITLE
Added lat/lon to unit_choices dropdown

### PR DESCRIPTION
**Pull request by Ethan-bradley**
This pull request added the latitude/longitude as an option in the unit_choices dropdown as mentioned in issue #28.

### CODE REVIEW

**Review by TangoYankee — APPROVED** [review]

File: woeip/apps/air_quality/models.py [hunk 1]
```diff
@@ -28,13 +28,13 @@ def __str__(self):
...
+    """A sensor is something that measures something, i.e., it produces a single
+    measurement value at a time.
     """
     name = models.CharField(max_length=256)
-    unit_choices = (('mg/m3', 'mg/m3'), ('ppm', 'ppm'), ('g/m3', 'g/m3'), ('PM10', 'PM10'), ('PM2.5', 'PM2.5'),
-                    ('μg/m3', 'μg/m3'))
-
+    unit_choices = (('mg/m3', 'mg/m3'), ('ppm', 'ppm'), ('g/m3', 'g/m3'), ('PM10', 'PM10'),
+                    ('PM2.5', 'PM2.5'),
+                    ('μg/m3', 'μg/m3'), ('latitude/longitude', 'latitude/longitude'))
```
Comment (r-b-g-b) [inline woeip/apps/air_quality/models.py]: Can I suggest naming it ``('latlon', 'Latitude/Longitude')``. ``latlon`` would have it match the field name: ``Data.latlon``

Comment (r-b-g-b) [reply woeip/apps/air_quality/models.py]: Also ``('PM2.5', 'PM2.5'), ('μg/m3', 'μg/m3'), ('latlon', 'Latitude/Longitude'))`` can all be on one line (120 character line limit)

File: woeip/apps/air_quality/models.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ class Sensor(models.Model):
     name = models.CharField(max_length=256)
     unit_choices = (('mg/m3', 'mg/m3'), ('ppm', 'ppm'), ('g/m3', 'g/m3'), ('PM10', 'PM10'),
                     ('PM2.5', 'PM2.5'),
-                    ('μg/m3', 'μg/m3'), ('latitude/longitude', 'latitude/longitude'))
+                    ('μg/m3', 'μg/m3'), ('latitude/longitude', 'latlong'))
```
Comment (r-b-g-b) [inline woeip/apps/air_quality/models.py]: I think the order should be flipped: first element is how it's stored in the db, second is a more human-readable version. So ``('latlon', 'Latitude/Longitude')``

**Review by r-b-g-b — CHANGES_REQUESTED** [review]

File: woeip/apps/air_quality/models.py (hunk 2 above)
Comment (TangoYankee) [reply woeip/apps/air_quality/models.py]: [correct](https://docs.djangoproject.com/en/2.1/ref/models/fields/#choices)

**Review by r-b-g-b — APPROVED** [review]

**Pull request merged**